modules/installation-azure-service-principal.adoc

```diff
@@ -177,87 +173,41 @@ output. You need these values during {product-title} installation.
 +
 [source,terminal]
 ----
-$ az ad sp create-for-rbac --role Contributor --name <service_principal> <1>
+$ az ad sp create-for-rbac --role Contributor --name <service_principal> \ <1>
+  --scopes /subscriptions/53b8f551-f0fc-4bea-8cba-6d1fefd54c8a
```

Comment:
could you help hide 53b8f551-f0fc-4bea-8cba-6d1fefd54c8a, this is a valid subscriptions QE is using.